### PR TITLE
Fixed CI

### PR DESCRIPTION
**Pull request by gnought**
We switch to Github Actions for our CI pipeline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # aedes-packet
-[![Build Status](https://travis-ci.org/mcollina/aedes-packet.svg?branch=master)](https://travis-ci.org/mcollina/aedes-packet)
+![](https://github.com/moscajs/aedes-packet/workflows/ci/badge.svg)
 [![Dependencies Status](https://david-dm.org/mcollina/aedes-packet/status.svg)](https://david-dm.org/mcollina/aedes-packet)
```
Comment (robertsLando) [inline README.md:3]: You forgot to fix the link here

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 [![devDependencies Status](https://david-dm.org/mcollina/aedes-packet/dev-status.svg)](https://david-dm.org/mcollina/aedes-packet?type=dev)
```
Comment (robertsLando) [inline README.md:4]: Also here

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (gnought) [reply README.md:4]: I leave it for another PR https://github.com/moscajs/aedes-packet/pull/8

File: README.md (hunk 1 above)
Comment (gnought) [reply README.md:3]: I leave it for another PR https://github.com/moscajs/aedes-packet/pull/8

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**